publications-2019.bib

```diff
@@ -1253,4 +1253,61 @@ @misc{engwer2019dynamic
     primaryClass={math.NA}
 }
 
+
+@article{Boddu2019,
+  doi = {10.1007/s42493-019-00027-z},
+  url = {https://doi.org/10.1007/s42493-019-00027-z},
+  year = {2019},
+  month = oct,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {1},
+  number = {4},
+  pages = {299--317},
+  author = {Vishal Boddu and Denis Davydov and Bernhard Eidel and Paul Steinmann},
+  title = {Cutoff-Based Modeling of Coulomb Interactions for Atomistic-to-Continuum Multiscale Methods},
+  journal = {Multiscale Science and Engineering}
+}
+
+@article{Bonito2019,
+  doi = {10.1137/18m1169278},
+  url = {https://doi.org/10.1137/18m1169278},
+  year = {2019},
+  month = jan,
+  publisher = {Society for Industrial {\&} Applied Mathematics ({SIAM})},
+  volume = {57},
+  number = {3},
+  pages = {973--996},
+  author = {Andrea Bonito and Alan Demlow},
+  title = {A Posteriori Error Estimates for the Laplace--Beltrami Operator on Parametric $C^2$ Surfaces},
+  journal = {{SIAM} Journal on Numerical Analysis}
+}
+
+@incollection{Carreo2019,
+  doi = {10.1007/978-3-030-22750-0_68},
+  url = {https://doi.org/10.1007/978-3-030-22750-0_68},
+  year = {2019},
+  publisher = {Springer International Publishing},
+  pages = {702--709},
+  author = {Amanda Carre{\~{n}}o and Antoni Vidal-Ferr{\`{a}}ndiz and Damian Ginestar and Gumersindo Verd{\'{u}}},
+  title = {A Matrix-Free Eigenvalue Solver for the Multigroup Neutron Diffusion Equation},
+  booktitle = {Lecture Notes in Computer Science}
+}
+
+@article{Vassaux2019,
+  doi = {10.1002/adts.201800168},
+  url = {https://doi.org/10.1002/adts.201800168},
+  year = {2019},
+  month = feb,
+  publisher = {Wiley},
+  volume = {2},
+  number = {5},
+  pages = {1800168},
```

Comment:
this is the DOI, not pages (the pages are just 1-6), though for some reason the journal put this information at the bottom of the page.